libs/platform/auth/src/components/login/login.component.ts

```diff
@@ -0,0 +1,317 @@
+import { resolve } from '@spryker-oryx/di';
+import { ContentMixin, defaultOptions } from '@spryker-oryx/experience';
+import { I18nService } from '@spryker-oryx/i18n';
+import { RouterService } from '@spryker-oryx/router';
+import { PasswordVisibilityStrategy } from '@spryker-oryx/ui/password';
+import {
+  FormAssociatedElement,
+  hydratable,
+  i18n,
+  subscribe,
+} from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { DirectiveResult } from 'lit/directive.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { createRef, ref } from 'lit/directives/ref.js';
+import { when } from 'lit/directives/when.js';
+import { catchError, EMPTY, map, Subject, switchMap, tap } from 'rxjs';
+import { DefaultAuthLoginStrategy } from './default-login.strategy';
+import { LoginOptions, LoginRequest } from './login.model';
+import { AuthLoginStrategy } from './login.strategy';
+import { styles } from './login.styles';
+
+@defaultOptions({
+  enableRememberMe: true,
+  redirectUrl: '/',
+  passwordVisibility: PasswordVisibilityStrategy.Mousedown,
+})
+@hydratable(['mouseover', 'focus'])
+export class AuthLoginComponent
+  extends ContentMixin<LoginOptions>(LitElement)
+  implements FormAssociatedElement
+{
+  static styles = styles;
+  static formAssociated = true;
+
+  @property() heading?: DirectiveResult | string;
+  @property() isLoading?: boolean;
+  @property() hasError?: boolean;
+  @property() emailName = 'email';
+  @property() passwordName = 'password';
+  @property() rememberMeName = 'rememberme';
+
+  @state() protected isDisabled = false;
+
+  #internals =
```

Comment:
But is it working in Safari?

Reply:
yes
<img width="585" alt="image" src="https://user-images.githubusercontent.com/3644678/222424902-ce1368d8-db0b-453a-a48c-a13e64797400.png">

Reply:
Actually I had to drop it because storybook does not support this syntax in decorated classes 🤷